apps/meteor/app/livechat/server/api/v1/webhooks.ts

```diff
@@ -69,10 +71,11 @@ API.v1.addRoute(
 
 			try {
 				Livechat.logger.debug(`Testing webhook ${webhookUrl}`);
-				const response = HTTP.post(webhookUrl, options);
+				const request = await fetch(webhookUrl, options);
+				const response = await request.text();
```

Comment:
If we are expecting a text response, shouldn't we use text within the `Accept` header instead of `application/json`?